src/main/java/com/airlines/model/Airplane.java

```diff
@@ -1,112 +1,66 @@
 package com.airlines.model;
 
 import java.time.LocalDate;
+import java.util.Objects;
 
 public class Airplane {
-    private Long id;
-    private String codeName;
-    private String model;
-    private LocalDate manufactureDate;
-    private int capacity;
-    private int flightRange;
-    private CrewMember crew;
-
-    public Airplane() {
-    }
-
-    public Airplane(Long id, String codeName, String model, LocalDate manufactureDate, int capacity, int flightRange, CrewMember crew) {
-        this.id = id;
-        this.codeName = codeName;
-        this.model = model;
-        this.manufactureDate = manufactureDate;
-        this.capacity = capacity;
-        this.flightRange = flightRange;
-        this.crew = crew;
+    private final Long id;
+    private final String codeName;
+    private final String model;
+    private final LocalDate manufactureDate;
+    private final int capacity;
+    private final int flightRange;
+    private final CrewMember crewMember;
```

Comment:
why you think Airplane has only one CrewMember? 
Your entity doesn't match to SQL database scheme